.github/workflows/site-refresh.yml

```diff
@@ -2,22 +2,29 @@ name: Refresh site in Surge
 
 on:
   schedule:
-      - cron: '*/15 * * * *'
+    - cron: "*/15 * * * *"
 jobs:
   refresh:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
+      - name: Checkout source code
+        uses: actions/checkout@v1
+      - name: Set up Node 10.x
```

Comment:
Version 12 will start being lts in a few days this October. We could do an issue to upgrade to v12 node.

Reply:
:+1: